### PR TITLE
modified version by issue-15

### PR DESCRIPTION
**Pull request by RushikeshJachak**
rushikesh jachak

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -20,6 +20,12 @@ <h1 class="page-heading center">Contributors</h1>
       </a>
       <a href="https://www.quora.com/profile/Sushant-Joshi-5">
         <i class="fa fa-quora"></i>
+        <li>
+      <a href="https://github.com/RushikeshJachak"> 
+        Rushikeshj
+      </a>
+
```
Comment (RushikeshJachak) [inline _layouts/home.html]: i added li tags in last  updated version

[the remark this answers is not in the document]
Comment (RushikeshJachak) [reply _layouts/home.html]: i added li tags in last  updated version

[the remark this answers is not in the document]
Comment (RushikeshJachak) [reply _layouts/home.html]: "added quora link" -->updated one.

**Comment by sushant-j** [comment]
Closing this as duplicate of #47

**Pull request closed without merging**